src/func/Microsoft.Health.Fhir.Ingest.Host/IomtConnectorFunctions.cs

```diff
@@ -12,45 +12,50 @@
 using Microsoft.Azure.EventHubs;
 using Microsoft.Azure.WebJobs;
 using Microsoft.Azure.WebJobs.Extensions.Http;
-using Microsoft.Extensions.Logging;
-using Microsoft.Health.Common.Telemetry;
 using Microsoft.Health.Fhir.Ingest.Data;
 using Microsoft.Health.Fhir.Ingest.Host;
 using Microsoft.Health.Fhir.Ingest.Telemetry;
 using Microsoft.Health.Fhir.Ingest.Template;
 
 namespace Microsoft.Health.Fhir.Ingest.Service
 {
-    public static class IomtConnectorFunctions
+    public class IomtConnectorFunctions
     {
+        private readonly ITelemetryLogger _logger;
+
+        public IomtConnectorFunctions(ITelemetryLogger logger)
+        {
+            _logger = logger;
+        }
+
         [FunctionName("MeasurementCollectionToFhir")]
-        public static async Task<IActionResult> MeasurementCollectionToFhir(
+        public async Task<IActionResult> MeasurementCollectionToFhir(
             [HttpTrigger(AuthorizationLevel.Function, "post", Route = null)] HttpRequest req,
             [Blob("template/%Template:FhirMapping%", FileAccess.Read)] string templateDefinition,
-            [MeasurementFhirImport] MeasurementFhirImportService measurementImportService,
-            ILogger log)

```

Comment:
Just curious why we can't ingest the logger instance via method parameter anymore and have to remove the static modifier?

Reply:
Either should work. 

I was reading some Azure Functions examples that weren't using the parameter and instead doing DI in the constructor, so I updated it to match the example.

https://docs.microsoft.com/en-us/azure/azure-monitor/app/azure-functions-supported-features
https://docs.microsoft.com/en-us/azure/azure-functions/functions-monitoring?tabs=cmd